packages/web-app-files/src/components/FileLinkSidebar.vue

```diff
@@ -25,7 +25,11 @@
             {{ $_addButtonLabel }}
           </oc-button>
         </div>
-        <p v-else class="oc-mt-s" v-text="noResharePermsMessage" />
+        <p
+          v-else
+          class="files-links-no-reshare-permissions-message oc-mt-s"
```

Comment:
Please use a `data-test-` attribute for all new selectors. In this case `data-test-id="xyz"`.